ui/pages/token-allowance/token-allowance.js

```diff
@@ -58,7 +63,7 @@ export default function TokenAllowance({
   data,
   isSetApproveForAll,
   isApprovalOrRejection,
-  customTxParamsData,
```

Comment:
Is it okay to stop using the `customTxParamsData` that is being passed into the component?

Reply:
right, okay, and I see how the customdata that was being passed in is only used in the `ConfirmApproveContent` component